gslib/tests/test_acl.py

```diff
@@ -83,7 +84,7 @@ def test_set_invalid_acl_object(self):
                             return_stderr=True,
                             expected_status=1)
     if self._use_gcloud_storage:
-      error_text = 'JSONDecodeError'
+      error_text = 'invalid JSON'
```

Comment:
Do you want to review this? https://github.com/GoogleCloudPlatform/gsutil/pull/1722

Reply:
Sure!